components/Trailer.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <div class="video">
+    <youtube
+      ref="youtube"
+      player-width="!fixed"
```

Comment:
the width isn't coming out right on large screens, I did find there is another library which supports styling better if the styles don't workout we can use it [vue-youtube](https://www.npmjs.com/package/vue-youtube)